src/main.rs

```diff
@@ -60,6 +60,10 @@ enum Cmd {
         /// Try to combine channels per segment group
         #[clap(short, long)]
         super_mechanisms: bool,
+        /// Change catalogue prefix.
```

Comment:
I do not think we need an option; we can just always use `local` for our compiled mechs.
Or even less prone to produce collisions: name it after the model.

Reply:
Shall I leave it in for the moment then another PR to remove it?

Reply:
Leave it, maybe further use shows we actually should have it.